services/net/indexing/IndexingManager.cs

```diff
@@ -390,7 +390,7 @@ private async Task<ContentModel> UnpublishContentAsync(ContentModel content)
             content = await this.Api.UpdateContentAsync(content) ?? throw new InvalidOperationException($"Content failed to update. ID:{content.Id}");
         }
 
-        var document = new DeleteRequest<ContentModel>(content, this.Options.PublishedIndex, content.Uid);
+        var document = new DeleteRequest<ContentModel>(content, this.Options.PublishedIndex, content.Id);
```

Comment:
I think this is why.  I won't know until I can test in Openshift.  Locally it all appears to work.